src/gwt/tools/i18n-helpers/commands.cmd.xml/sample_commands.cmd.xml

```diff
@@ -0,0 +1,125 @@
+<!--
```

Comment:
What is this file for? Do we need to check it in?

Reply:
I think this was for testing but after a quick glance I believe I didn't use it in the tests. I'll look more closely later today